fe/fe-core/src/main/java/com/starrocks/connector/ConnectorTableInfo.java

```diff
@@ -34,7 +34,7 @@ protected ConnectorTableInfo(Set<MvId> relatedMaterializedViews) {
 
     public void updateMetaInfo(ConnectorTableInfo tableInfo) {
         if (relatedMaterializedViews == null) {
-            relatedMaterializedViews = Sets.newHashSet(tableInfo.relatedMaterializedViews);
+            relatedMaterializedViews = Sets.newConcurrentHashSet(tableInfo.relatedMaterializedViews);
```

Comment:
why not make `relatedMaterializedViews` type as ConcurrentSet?

Reply:
The `relatedMaterializedViews`  is already ConcurrentHashSet, it's build from ConnectorTableInfo.Builder, this is just defensive programming